test/client/fixtures/sandbox/node/window-test.js

```diff
@@ -11,6 +11,31 @@ test('window.onerror setter/getter', function () {
     strictEqual(getProperty(window, 'onerror'), handler);
 });
 
+module('timers');
+
+asyncTest('setTimeout script argument processing', function () {
```

Comment:
What is the good of it? We just override native method, there is no difference as we call it.